lambda_functions/ml_service/GetKeyPhrases.py

```diff
@@ -45,14 +73,20 @@ def get_phrases(event, context):
         logger.error("Language Code not supported!")
         return []
 
+    response = {}
     try:
-        response = comprehend.detect_key_phrases(Text=text, LanguageCode=LanguageCode)
-    except ClientError as e:
-        logger.error("Received error: %s", e, exc_info=True)
-        return []
-    except ParamValidationError as e:
-        logger.error("The provided parameters are incorrect: %s", e, exc_info=True)
-        return []
+        response = post_TopicalPageRank(LanguageCode, text)
+    except Exception as e:
+        logger.error("TopicalPageRank error: %s", e, exc_info=True)
+    if not "KeyPhrases" in response:
```

Comment:
Is this meant as a backup, in case the model cannot provide any keyphrases?

Reply:
Yes, this is meant as back up, if the model container is not running.